docs/skill_development/skill-structure/README.md

```diff
@@ -0,0 +1,200 @@
+---
+description: Exploring the foundational components of a basic Neon Skill.
+---
+
+# Skill Structure
+
+```shell
+$ ls -l
+total 20
+drwxr-xr-x 3 kris kris 4096 Oct  8 22:21 dialog
+-rw-r--r-- 1 kris kris  299 Oct  8 22:21 __init__.py
+-rw-r--r-- 1 kris kris 9482 Oct  8 22:21 LICENSE
+-rw-r--r-- 1 kris kris  283 Oct  8 22:21 README.md
+-rw-r--r-- 1 kris kris  642 Oct  8 22:21 settingsmeta.yaml
+drwxr-xr-x 3 kris kris 4096 Oct  8 22:21 vocab
+```
+
+We will look at each of these in turn.
+
+## `vocab`, `dialog`, and `locale` directories
+
+The `dialog`, `vocab`, and `locale` directories contain subdirectories for each spoken language the skill supports. The subdirectories are named using the [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag) for the language. For example, Brazilian Portugues is 'pt-br', German is 'de-de', and Australian English is 'en-au'.
```

Comment:
Preferred structure is now:
locale -> language -> (dialog, vocab, regex, intent)

Note that everything under "language" can be in any structure but is generally organized by resource type